### ISSUE TITLE
Cannot equip a gear

### ISSUE THREAD

**Issue by Gradius2** [DarkstarProject/darkstar#6379]
Well, this bug was present on previous version too.

I'm now using the last one as I recompiled and updated DB, scripts, etc, today.

The bug is still there.

I just cannot equip Epaminondas's Ring, is a lvl 99 for ALL jobs:
https://www.ffxiah.com/item/26214/epaminondass-ring

No matter to which job I change, I'm just unable to equip it (or use it).

Please, check it out.

**Comment by Gradius2** [comment]
Update, another BUG...

Cannot use food!  I'm trying to use:
https://www.ffxiah.com/item/5894/prm-mn-stewpot

And it says "You cannot attack that target".  lol

**Comment by kaincenteno** [comment]
not a bug, most likely those items are not implemented (several of the items are not)

**Comment by EpicTaru** [comment]
Yeah, just checked, not finding those ItemIDs anywhere (scripts folder/sql folder/src folder) on DSP master.  The sales history of that food item began in 2019 and that armor item is level 99.  As Kain pointed out, many things post lvl75 content aren't yet coded on DSP master. `¯\_(ツ)_/¯`

**Comment by Gradius2** [comment]
I see.  It's weird since I'm playing as lvl 99 and have a lot 119 lvl gear on my char.  So I thought it would works.

Thanks for clarification.
